src/routes/guides/getting-started-with-solid/building-ui-with-components.mdx

```diff
@@ -315,7 +315,7 @@ There are a couple terms worth learning when we talk about how components relate
 
 ### Passing props between components
 
-Components can communicate with their child components by passing properties, which are commonly called `props`. To demonstrate the power of `props`, let's customize our bookshelf by adding our name to it! In the example below, we update our app to display "Solid's Bookshelf," but you should use your own name.
+Components can communicate with their child components by passing properties called `props`. To demonstrate the power of `props`, let's customize the bookshelf by adding your name! In the example below, we update the app to display "Solid's Bookshelf," but you should use your name.
```

Comment:
@Jutanium Do we have a standardized language we are using? 

Customize (American) vs Customise (British)

Reply:
> passing properties called `props`

Makes it feel like mentioning `props` is redundant. In reality, highlighting that they're also called `props` is a helpful distinction to newcomers.

Could we maybe try:

```suggestion
Components can communicate with their child components by passing properties, often shortened to `props`. To demonstrate the power of `props`, let's customize the bookshelf by adding your name! In the example below, we update the app to display "Solid's Bookshelf," but you should use your name.
```

Or some variation of such?

Reply:
Ohhh, this sounds better! I like it.

Reply:
@Hopelezz we're using American English - just added to the writing guide!
@kaleem68 if you could review @crutchcorn's suggestion and hit "Commit suggestion," that would be awesome!